mods/cnc/maps/gdi01/map.yaml

```diff
@@ -424,9 +424,6 @@ Rules:
 		-SpawnMPUnits:
 		-MPStartLocations:
 		-CrateSpawner:
-		PlayMusicOnMapLoad:
```

Comment:
I can create an upgrade rule that makes this vanish, but I am not sure if this will help mappers as then their music will be suddenly gone.